### ISSUE TITLE
Feedback High shutter speeds 1/1000+ accuracy with 1.7t2

### ISSUE THREAD

**Issue by afrankra** [Reveni-Matt/Camera-Tester#3]
Dear Matt,

Thank you for the 1.7t2 Firmware.
With:

> -       Grounded
> -       Canon 500n no lens 1/500s -> 1/2000s
> -       High speed of 500n corroborated with Second laser tester for high speeds (reduces side light bleed)

Prior to it 1.7t2, with 1.6 or earlier, readings with the 500n on vertical shutter mode would read out noise (no useful reading) at 1/1000+ shutter speeds.
The noise at lower speeds is not that significant, since it gets masked by the long exposure time. Meaning that the measurement error is not a significant part of the exposure time. 

The 1.7t2 creates useful readings  at 1/1500+  on the central sensor and diagonals (my right diagonal still has issues sometimes reading anything, but moving the head around to a diff position in the 35mm frame shows that it does work).

The readings are nevertheless not quite accurate. I think due to light bleed or light intensity.
Moving the sensor head backwards from the 35mm frame (unseated with around 5mm distance from film plane) increases the accuracy to near perfection for 1/2000 : (1/2050-1/1950).
Not sure if using a lens (Dont have one atm) would improve the readings by reduce the amount of light.

Just wanted to give the feedback. Perhaps the changes of 1.7t2 could be further fine tuned, or maybe the user can select the panel light intensity if necesary.

**Comment by Reveni-Matt** [comment]
Light intensity is calibrated for, at the start of the calibration cycle, the sensors are tested to find their trip point.  This helps me determine at what part during the cover/uncover phase they will flip from low to high and compensate for that.

On 1.7T2 I am seeing good speeds on all sensors at 1/2000 and even 1/8000 on my Sony Alpha 9.

Almost any reduction in panel brightness will cause issues with triggering the sensors.  Even lowering it one stop can make it unreliable to trigger on some cameras.

Using a lens doesn't work because the angle the light hits the film plane is too shallow and the narrow FOV of the detectors prevents the left and right detectors from triggering at those angles.  The light needs to be coming in almost dead straight.

> Moving the sensor head backwards from the 35mm frame (unseated with around 5mm distance from film plane) increases the accuracy to near perfection for 1/2000 : (1/2050-1/1950).

Doing this usually causes the numbers to increase a little bit, it can vary.  Generally I find it less reliable.

See attached pictures.

![2024-07-16 16 03 45](https://github.com/user-attachments/assets/f470f052-af35-49fb-886e-79b18204022b)


![2024-07-16 16 04 25](https://github.com/user-attachments/assets/f531d770-488b-4490-9491-a23e96781793)

**Comment by afrankra** [comment]
During the calibration process that determines the tripping points, at what distance must the sensor head be from the light panel?

**Comment by Reveni-Matt** [comment]
The sensor should be placed against the LED panel surface for calibration.

**Comment by Reveni-Matt** [comment]
I added a new firmware V1.8T1, you can try it out.  Moving the head around so that two detectors are well exposed and the third is obscured now works as an alternative method in cameras which have issues getting all 3 to fire.

**Comment by afrankra** [comment]
Hi Matt. Thank you for the Update. I will do test my previously problematic cameras tomorrow.

>Moving the head around so that two detectors are well exposed and the third is obscured now works as an alternative method in cameras which have issues getting all 3 to fire.

This was exactly my issue, some cameras do not have a perfectly framed lightpath from film plane to light source (that is the lense job!) hopefully this update gives more flexibility! Thank you for your help! I will report soon my results.

**Comment by afrankra** [comment]
Hi Matt,
Firmware v1.8 is great at 1/2000s + . I think this issue can be closed. 
I can hint at some interesting things tho: 

1. Some plane shutter cameras have a smaller lens opening than the film plane opening:
![image](https://github.com/user-attachments/assets/f33a2988-ec0d-4aa4-8dd1-2dc454b128b6)
![image](https://github.com/user-attachments/assets/06df2cae-4ff5-4d3e-bbfb-da9e813ec0cd)
This makes so that the edge sensors dont trigger properly. Just an FYI. 

2. It takes 1/8s or slower speeds to accurately measure the T stop on the Exposure modes, which leads to wrong EV estimates.
I was very confused as to why the Lens Aperture test could accurately measure the T stop, but when using auto exposure tests, the T stop would be 1 to 1 1/2 stops slower . Only at 1/8s or slower could I get the T stop to be measured correctly in the auto exposure modes. 
Not sure if this is an integration limitation or if it can be compensated for.

**Comment by Reveni-Matt** [comment]
Yes I've heard this issue from several people in the last 2 or 3 weeks.  It took shipping over 200 units before someone had this problem it seems.  The issue is due to the narrow lens mount preventing enough light from reaching the detectors: 

<img width="1297" alt="detector openings render" src="https://github.com/user-attachments/assets/4dc83189-6519-429f-9eeb-132105e4bc83">
<img width="1066" alt="sensors obscured issue" src="https://github.com/user-attachments/assets/6e29f5cf-234b-463b-a24c-6979102aaf51">

I'm going to experiment with angled sensor holes to try to improve the performance with small lens openings.  Maybe even get it to work with a lens attached, although I don't think that will do anything to improve performance.

Regarding your second part, can you send some pictures of the results screen?  I get pretty good aperture readings at high speeds.

![2024-08-05 12 56 14](https://github.com/user-attachments/assets/ef648eaf-e37f-4aae-8405-c56146cc501b)
![2024-08-05 12 55 20](https://github.com/user-attachments/assets/3b3cb8d8-96f0-4c11-8622-0dcc142a9d06)
